pkgs/tools/text/papertrail/default.nix

```diff
@@ -21,6 +21,8 @@ in stdenv.mkDerivation {
 
   passthru.updateScript = bundlerUpdateScript "papertrail";
 
+  passthru.tests.simple = callPackage ./test.nix {};
```

Comment:
Please use `testers.testVersion`

Reply:
Thanks, wasn't aware of `testers.testVersion`. Done.